### PR TITLE
AUTH-29 Build scripts for cert-manager

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.redhat.com/browse/AUTH-29

This Pull Request introduces CI jobs definitions as well as the release jobs definitions for https://github.com/openshift/cert-manager-operator

### CODE REVIEW

File: ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-master.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          cpu: 100m
+    workflow: ipi-gcp
+- as: e2e-agnostic-upgrade
+  steps:
+    cluster_profile: azure4
+    workflow: openshift-upgrade-azure
+- as: e2e-aws-single-node
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws-single-node
```
Comment (stlaz) [inline ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-master.yaml:72]: we probably don't need these yet, we're not a part of the default payload and we don't have any operator's e2e tests

Comment (slaskawi) [reply ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-master.yaml]: Removed.

**Comment by slaskawi** [comment]
@stlaz @s-urbaniak @openshift/openshift-team-developer-productivity-test-platform Could I ask to review this one? 

The only error remaining happens in `verify`. However, this goal works fine in box so I'd appreciate any hints in investigating it.

**Comment by hongkailiu** [comment]
> @stlaz @s-urbaniak @openshift/openshift-team-developer-productivity-test-platform Could I ask to review this one?
> 
> The only error remaining happens in `verify`. However, this goal works fine in box so I'd appreciate any hints in investigating it.

Have you tried to reproduce it with the builder image?

**Comment by slaskawi** [comment]
> Have you tried to reproduce it with the builder image?

@hongkailiu Excellent advice, thank you! The fix is out there: https://github.com/openshift/cert-manager-operator/pull/12

**Pull request merged**